samples/GetUserProfile.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+# Get a users display name and avatar
+# Args: host:port username password user_id
+# Error Codes:
+# 2 - Could not find the server.
+# 3 - Bad URL Format.
+# 4 - Bad username/password.
+# 11 - Wrong room format.
+# 12 - Couldn't find room.
+
+
+import sys
+sys.path.insert(0, "../") # add ../ to PYTHONPATH
+
+from matrix_client.client import MatrixClient, User
+from matrix_client.api import MatrixRequestError
+from requests.exceptions import MissingSchema
+from getpass import getpass
+
+host = ""
+username = ""
+room = ""
```

Comment:
Ah, these aren't constants.

Generally empty variables in python are set to `None`, or just not declared until you set them. Don't forget that blocks don't create new scopes, e.g. after the following bit of code `host` will be visible to the rest of the function:

```python
if whatever:
    host = "foo"
else:
    host = "bar"
```